docs/api/covidcast_changelog.md

```diff
@@ -17,98 +23,8 @@ we **strongly recommend** that anyone using the API to download data regularly
-- The split of the Dukes and Nantucket, Massachusetts FIPS codes into the two sub-counties is now done with a more precise population proportion,
-- Some counts are reported in the Utah territories JHU UIDs 84070015-84070020. These are now aggregated properly into the state megaFIPS 49000, which will be aggregated in state-level information.
-
-### `indicator-combination`
-
-#### 3 June 2020
-
-Standard errors are now included in the `nmf_day_doc_fbc_fbs_ght` signal for all geo levels and dates, representing the estimated uncertainty in this signal. This uncertainty comes because the signal is a combination of other signals based on survey estimates or other estimates with margins of error.
-
-* `nmf_day_doc_fbc_fbs_ght`
-  * all geo levels
-    * all dates
-
-#### 12 October 2020
-
-The 10 October 2020 issue of all `indicator-combination` deaths signals has been removed from the API. These signals are primarily constructed of USAFacts data, whose 10 October 2020 issue was discovered to be corrupt on 11 October and repaired on 12 October. Subsequent issues have adequate coverage of all regions and dates included in the 10 October issue, so this change only affects forecasters who intend to pull training data with an `as_of` or `issues` parameter set to 20201010.
-
-### `quidel`
-
-#### 22 October 2020
-We went from a custom geo mapping file (for aggregating from zip->(county, msa, hrr, state)) to a central geo file based on rigorously sourced US census data.
-
-### `safegraph`
-
-#### 19 April 2021
-
-The Safegraph social distancing metrics are no longer being updated. Weekly patterns are still available.
-
-#### 3 November 2020
-We went from a custom geo mapping file (for aggregating from county->state) to a central geo file based on rigorously sourced US census data.
-
-### `usa-facts`
-
-#### 3 November 2020
-We went from a custom geo mapping file (for aggregating from county->(msa, hrr, state)) to a central geo file based on rigorously sourced US census data.
-
-#### 12 October 2020
-
-The 10 October 2020 issue of all `usa-facts` deaths signals has been removed from the API. The file for deaths provided by USAFacts on 10 October included case data instead. The resulting spurious 100× increase in magnitude of COVIDcast `usa-facts` deaths signals was noticed on 11 October and repaired on 12 October. Subsequent issues have adequate coverage of all regions and dates included in the 10 October issue, so this change only affects forecasters that intend to pull training data with an `as_of` or `issues` parameter set to 20201010.
+For current information on how up-to-date our public health and behavior data
+is, how much of the country is covered by our public health and behavior data,
+and the data versioning effects present in our public health and behavior data,
+please see Delphi's [Indicator Status Overview
+page](https://delphi.cmu.edu/covidcast/indicator-status/).
```

Comment:
Open to just getting rid of this page entirely, this blurb just seemed too long to paste at the top of every individual indicator changelog.